packages/appstore/src/components/account/account.scss

```diff
@@ -0,0 +1,126 @@
+.account__container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0.4rem 0;
+    isolation: isolate;
+    position: relative;
+    width: 24rem;
+    height: 7.2rem;
+    background: $color-grey-2;
+    border-radius: 0.6rem 0.6rem 0 0;
+    @include mobile {
+        padding: 0.8rem;
+        gap: 0.4rem;
+        isolation: isolate;
+        width: 22rem;
+        height: 6.2rem;
+        background: linear-gradient(0deg, $color-grey-2, $color-grey-2), #ffffff;
```

Comment:
why not `background-color: $color-grey-2` ?

Reply:
@mahdiyeh-fs , I took the style attribute from figma design , Should I change ?

Reply:
yes, please